Signal/src/Models/TSMessageAdapaters/TSPhotoAdapter.m

```diff
@@ -112,7 +113,18 @@ - (void)performEditingAction:(SEL)action
     }
 
     if (action == @selector(copy:)) {
-        UIPasteboard.generalPasteboard.image = self.image;
+        OWSAssert(self.attachment.filePath.length > 0);
```

Comment:
Because this code is non-obvious, it might be worth leaving a comment as to why we're losing the much more concise version of the above.